package.json

```diff
@@ -78,8 +79,10 @@
     "fixtures:load": "cozy-fixtures load tests/fixtures",
     "fixtures:loader": "coffee tests/fixtures/loader.coffee",
     "lint": "npm-run-all 'lint:*'",
-    "lint:coffeelint": "coffeelint --file coffeelint.json --quiet ./server/**/*.coffee",
-    "lint:jshint": "coffee-jshint --options node ./server/**/*.coffee",
+    "lint:server": "npm-run-all 'lint:server:*:*'",
+    "lint:server:code:coffeelint": "./node_modules/.bin/coffeelint --file coffeelint.json  server.coffee ./server/*.coffee ./server/**/*.coffee",
```

Comment:
I'm not sure that we need to be too much precise with tasks namespace. What for a `lint:server:coffee` (which lint codebase and tests) and `lint:server:js`(for the js part)?